class-gwiz-gf-feed-forge.php

```diff
@@ -442,9 +443,22 @@ public static function process_entry_feeds( $entries, $feeds, $form_id ) {
 					continue;
 				}
 
+				$addon           = $addons[ $feed['addon_slug'] ];
+				$processed_feeds = $addon->get_feeds_by_entry( $entry_id );
+				if ( is_array( $processed_feeds ) && in_array( $feed['id'], $processed_feeds ) && $reprocess_feeds ) {
+					$all_processed_feeds = gform_get_meta( $entry_id, 'processed_feeds' );
+
+					// Remove feed id from processed feeds
+					$key = array_search( $feed['id'], $processed_feeds );
+					unset( $processed_feeds[ $key ] );
+					$all_processed_feeds[ $feed['addon_slug'] ] = array_values( $processed_feeds );
+
+					gform_update_meta( $entry_id, 'processed_feeds', $all_processed_feeds, $feed['form_id'] );
+				}
```

Comment:
Thoughts on extracting this out into a new method? e.g. `public static function clear_processed_feeds( $form_id $entry_id, $feed_ids )`?